checkov/cdk/checks/typescript/LambdaEnvironmentEncryptionSettings.yaml

```diff
@@ -0,0 +1,66 @@
+metadata:
+  version: 0.2
+  approach: define failing
+  id: CKV_AWS_173
+  name: Check encryption settings for Lambda environmental variable
+  category: Encryption
+  framework: cdk
+scope:
+  languages:
+    - typescript
+definition:
+  or:
+    - pattern: |
+        new $PACKAGE.aws_lambda.Function(<ANY>, <ANY>, { <ANY>, environment: { <ANY>, $ARG1:'$ARG2', <ANY> }, <ANY> } )
+      conditions:
+        - not_pattern: |
+            new $PACKAGE.aws_lambda.Function(<ANY>, <ANY>, { <ANY>, environment: { <ANY>, $ARG1:'$ARG2', <ANY> }, <ANY>, environmentEncryption: $KMS, <ANY> })
+        - not_pattern: |
+            new $PACKAGE.aws_lambda.Function(<ANY>, <ANY>, { <ANY>, environmentEncryption: $KMS, <ANY>, environment: { <ANY>, $ARG1:'$ARG2', <ANY> }, <ANY> })
```

Comment:
Did you open a ticket to support dict elements not-considering their order?

Reply:
nope, that why i did it like python did...